rts/Lua/LuaConstCMD.cpp

```diff
@@ -60,11 +71,81 @@ bool LuaConstCMD::PushEntries(lua_State* L)
 
 #define PUSH_CMD(cmd) LuaInsertDualMapPair(L, #cmd, CMD_ ## cmd);
 
-	/*** @field CMD.STOP 0 */
+	/***
+	 * @field CMD.STOP 0
+	 *
+	 * Stop the current action.
+	 *
+	 * For factories, this will cancel the new unit orders queue.
+	 * For units, this will cancel the current command and queue.
+	 *
+	 * Accepts no parameters.
+	 *
+	 * *Note:* This is actually a noop, but clears the queue when `shift` isn't used since
+	 * that's the standard behaviour when `shift` isn't used for all actions.
```

Comment:
I would put it like this.
```suggestion
	 * Clears target and stops movement.
	 *
	 * Does NOT actually clear the order queue on its own,
	 * but is typically used that way because all commands
	 * clear the queue when not enqueued (given without shift).
	 *
	 * Accepts no parameters.
```

Reply:
Rephrased it differently now. Thing here, is not sure how much user facing we should try to be. Still undecided tbh. I think your new description is nice, but it all depends if we want to write that kind of internal details there.

Only thing I'd change from this description is the first sentence, since it doesn't really say it clears the queue:

`Clears the unit's queue, current target and stops movement.`

or something like that.

Reply:
> the first sentence, since it doesn't really say it clears the queue

I think the second sentence clears it up pretty well.